examples/codegen/elixir_api/test/elixir_api_web/controllers/blocked_times_controller_test.exs

```diff
@@ -0,0 +1,32 @@
+defmodule ElixirApiWeb.BlockedTimesControllerTest do
```

Comment:
It's important to think about the 3 layers of tests that we have here to make sure we're not testing the shape of the data multiple times needlesly. The resource tests are generated. The view tests should be very dumb (are they even necessery?)

In the controller tests we could use some openApi magic to verify the responses. Also If we make the controllers dumb enough to not need any human changes then the tests could probably be fully generated too.

Then we would test logic in the core modules, and the controllers, resources data shapes would all come from openapi